sdk/eventhubs/azure-messaging-eventhubs/vcpkg.json

```diff
@@ -2,14 +2,7 @@
   "name": "azure-messaging-eventhubs",
   "version-string": "1.0.0",
   "dependencies": [
-    {
-      "name": "azure-core-cpp"
-    },
-    {
-      "name": "azure-core-amqp-cpp"
-    },
-    {
-      "name": "azure-storage-blobs-cpp"
-    }
+    "azure-core-amqp-cpp",
+    "azure-storage-blobs-cpp"
```

Comment:
This is an issue.
1) eventhubs should certainly not depend on storage blobs
2) Why aren't we listing core as a dependency? Is it coming transitively through the AMQP dependency?

Reply:
That is true - eventhubs depend on storage blobs.